executor.go

```diff
@@ -293,14 +293,22 @@ func (e *Executor) taskEventLoop() {
 		case Unhealthy:
 			unhealthy := false
 			e.stateUpdater.UpdateWithOptions(taskInfo.GetTaskID(), mesos.TASK_RUNNING, state.OptionalInfo{Healthy: &unhealthy, Message: &event.Message})
-		case FailedDueToUnhealthy, FailedDueToExpiredCertificate:
+		case FailedDueToUnhealthy:
 			unhealthy := false
 			info := state.OptionalInfo{Healthy: &unhealthy, Message: &event.Message}
 			e.stateUpdater.UpdateWithOptions(taskInfo.GetTaskID(), mesos.TASK_RUNNING, info)
 			log.WithFields(log.Fields{"TaskID": taskInfo.GetTaskID(), "Reason": event.Message}).Info("Killing task")
 			e.shutDown(taskInfo, cmd)
 			e.stateUpdater.UpdateWithOptions(taskInfo.GetTaskID(), mesos.TASK_FAILED, info)
 			return
+		case FailedDueToExpiredCertificate:
+			unhealthy := false
+			info := state.OptionalInfo{Healthy: &unhealthy, Message: &event.Message}
+			e.stateUpdater.UpdateWithOptions(taskInfo.GetTaskID(), mesos.TASK_RUNNING, info)
+			log.WithFields(log.Fields{"TaskID": taskInfo.GetTaskID(), "Reason": event.Message}).Info("Killing task")
```

Comment:
maybe add reason why task is killed?

Reply:
There already i in the Reason field, because of
```go
e.events <- Event{Type: FailedDueToExpiredCertificate, Message: "Certificate expired"}
```

Reply:
yeah but the one you pointed is sent to Marathon and the one I pointed is a local executor log